app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -72,6 +74,12 @@
     private String lastVisibleItemID;
 
     private int SPAN_COUNT=3;
+    private List<Contribution> contributions;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
```

Comment:
if all this does is call super then it is unnecessary and can be deleted

Reply:
Done